app/controllers/staff/activities_controller.rb

```diff
@@ -31,6 +31,19 @@ def show
     end
   end
 
+  def edit
+    @activity = Activity.find(id)
```

Comment:
I'm wondering if we want a custom action name here instead of edit/update. And if we do, how to fit it into our routes file, which is very unusual at the moment!!

Reply:
How about using a singular resource?

```ruby
resources :activities, only: [], concerns: [:transactionable, :budgetable, :disbursement_plannable] do
      resource :redaction, only: [:edit, :update], controller: :activity_redactions
      .. etc ..
end
```

Then you can move these actions into `ActivityRedactionsController` and paths like `/activities/:activity_id/redaction/edit` ?